### DISCUSSION TITLE
Add dependecies to debug session

### DISCUSSION THREAD

**Discussion by ulise** [camel-tooling/camel-dap-client-vscode#364] [Q&A] [answered]
Hi, ist there a way to add dependencies (like camel-sql) to the debug session?

**Comment by apupier** [comment]
I'm not sure to understand the question.

Do you mean when using the "Debug with Jbang" on a route which does not include a sql camel component?

it is always possible to use a Camel on SpringBoot or Camel on Quarkus and so providing dependencies in the pom.xml. In this case, you need to use launch configurations and tasks and cannot use the shortcut action.

**Reply by ulise** [reply, marked as answer]
Hi, sorry to be so unclear. I want to start a java route using "run and debug with jbang..." in vscode, but I need some extra dependencies.
  I found meanwhile, that I could add them to camel-jbang-run.properties, right? Many thanks.

**Reply by apupier** [reply]
Using this file is the way to achieve it when using the "Run and debug with JBang". For reference for other potential users reading these lines: https://camel.apache.org/manual/camel-jbang.html#_configuration
